server/constants.js

```diff
@@ -6,30 +6,30 @@ module.exports = {
- CONTRACT_ADDRESS : process.env.CONTRACT_ADDRESS || '0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85',
```

Comment:
Can we keep these defaults because it makes local development easier

Reply:
Reverted `constants.js`